assessment/utils/assessment_output.go

```diff
@@ -0,0 +1,69 @@
+/*
+	Copyright 2025 Google LLC
+
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+package utils
+
+import "github.com/GoogleCloudPlatform/spanner-migration-tool/spanner/ddl"
+
+type AssessmentOutput struct {
```

Comment:
I'm not sure this should be in utils as it is local to the assessment engine and reporting.

Is it required outside ?